Engine/EditorUI/Panel/InspectorSubpanel/PanelGameObject.cpp

```diff
@@ -45,10 +45,17 @@ PanelGameObject::PanelGameObject()
 
 void PanelGameObject::Render(GameObject* game_object)
 {
+	
 	if (game_object == nullptr)
 	{
 		return;
 	}
+	focused = ImGui::IsWindowFocused();
```

Comment:
Which Window is Focusing? Scene/Game?

Reply:
is the panel hierarchy window the one that get focus